packages/labs/react/src/create-component.ts

```diff
@@ -48,15 +48,25 @@ export type ReactWebComponent<
 // lifecycle methods or allow user to explicitly provide props.
 type ElementProps<I> = Partial<Omit<I, keyof HTMLElement>>;
 
-// Acceptable props to the React component. Omit keyof E from HTMLAttributes to
-// prefer provided event handler mapping over React's built-in event handler
-// props.
+// Child type compatible with both React and Preact.
```

Comment:
Is it worth adding some additional details/references for how/where this type was derived from?

Reply:
Done.